test/src/test/java/org/corfudb/CorfuTestParameters.java

```diff
@@ -111,7 +111,7 @@ public CorfuTestParameters(){
         NUM_ITERATIONS_VERY_LOW = TRAVIS_BUILD ? 1 : 10;
         NUM_ITERATIONS_LOW = TRAVIS_BUILD ?  10 : 100;
         NUM_ITERATIONS_MODERATE = TRAVIS_BUILD ? 100: 1000;
-        NUM_ITERATIONS_LARGE = TRAVIS_BUILD ? 1_000 : 10_000;
+        NUM_ITERATIONS_LARGE = TRAVIS_BUILD ? 1_000 : 15_000;
```

Comment:
Why is this value being modified?

Reply:
To use it in places where we want to write more entries than the size of segment.

Reply:
This changes the value for ALL the tests. This is NOT the place to change. Where is the segment size being set. Seems like a simple change would be to set the segment size to moderate rather than changing EVERY test.

Reply:
Then I'll just make it test variable instead. The segment size is not easily configurable right now.

Reply:
Or, you could make a NUM_ITERATIONS_VERY_LARGE. but I guess that should be 50k or 100k

Reply:
That's too large, some of the tests were timing out on 20k, i think.